assemblyline/common/custom.yara

```diff
@@ -451,10 +451,12 @@ rule code_ps1 {
         $ = /\$\w+[ \t]*=[ \t]*[^;^\n^|]+[;\n|]/ ascii wide
         $ = /\bfunction[ \t]+\w+[ \t]*\([^)]*\)[ \t\n]*{/i ascii wide
         $ = /\[char\][ \t]*(\d\d|0x[0-9a-f]{1,2})/i ascii wide
+        // This powershell regex is the same as the one in code/batch except it does not need -c or -command
+        $power = /(^|\n|@|&)\^?p(\^|%.+%)?o(\^|%.+%)?w(\^|%.+%)?e(\^|%.+%)?r(\^|%.+%)?s(\^|%.+%)?h(\^|%.+%)?e(\^|%.+%)?l(\^|%.+%)?l(\^|%.+%)?(\.(\^|%.+%)?e(\^|%.+%)?x(\^|%.+%)?e(\^|%.+%)?)?.+(-c|-command)?(\^|%.+%)?[ \t]/i
```

Comment:
I think you could replace the entire `\.exe.+(-c|-command)` section with a word boundary check unless the `[ \t]` is meaningfully weeding out false positives. All of it is optional or matches everything except for `[ \t]` and we care about if it matches rather than what is matched in this context.